packages/SwingSet/src/kernel/deviceManager.js

```diff
@@ -51,16 +50,19 @@ export default function makeDeviceManager(
     testLog,
   );
 
+  /**
+   * @typedef {Object} CapData
+   * @property {string} body
+   * @property {Array<string>} slots
+   */
+  /**
+   * @typedef {['ok', CapData]} VatInvocationResults
+   */
   /**
    * Invoke a method on a device node.
    *
-   * @param target  Kernel slot designating the device node that is the target
-   *    of the invocation
-   * @param method  A string naming the method to be invoked
-   * @param args  A capdata object containing the arguments to the invocation
-   *
-   * @return a VatInvocationResults object: ['ok', capdata]
-   *
+   * @param {[string, string, CapData]} deviceInvocation
```

Comment:
The descriptions of the three components of `deviceInvocation` need to be retained somewhere, just adding `[target, method, args]` to the comment would be sufficient.

Reply:
I've done the moral equivalent of this.